package/src/values/web/RNSkComputedValue.ts

```diff
@@ -1,9 +1,7 @@
-import type { DependencyList } from "react";
-
 import { RNSkReadonlyValue } from "./RNSkReadonlyValue";
 
 export class RNSkComputedValue<T> extends RNSkReadonlyValue<T> {
-  constructor(callback: () => T, dependencies: DependencyList) {
+  constructor(callback: () => T, dependencies: object[]) {
```

Comment:
Why this change? Doesn't `DependencyList` work well (`ReadonlyArray<any>`)?

Reply:
nice catch! `DependencyList` has been upgraded to `ReadonlyArray<unknown>`, which also is more precise.